sync/class.jetpack-sync-module-woocommerce.php

```diff
@@ -113,4 +118,115 @@ public function estimate_full_sync_actions( $config ) {
 	private function get_where_sql( $config ) {
 		return '1=1';
 	}
+
+	public function options_whitelist( $list ) {
+		return self::add_to_whitelist( $list, self::$wc_options_whitelist );
+	}
+
+	public function constants_whitelist( $list ) {
+		return self::add_to_whitelist( $list, self::$wc_constants_whitelist );
+	}
+
+	public function post_meta_whitelist( $list ) {
+		return self::add_to_whitelist( $list, self::$wc_post_meta_whitelist );
+	}
+
+	private static function add_to_whitelist( $whitelist, $values ) {
```

Comment:
In usage, above seems to use the new values as the first param, but this seems to expect them as the second?  It likely doesn't matter hugely, but may be nice to better sort.  Or just have the other methods call `array_merge()` directly -- it's simple enough.